api/apps/brand_partners/model.coffee

```diff
@@ -0,0 +1,83 @@
+#
+# Library of retrieval, persistance, validation, json view, and domain logic
+# for the "brand-partners" resource.
+#
+
+_ = require 'underscore'
+db = require '../../lib/db'
+async = require 'async'
+Joi = require 'joi'
+Joi.objectId = require 'joi-objectid'
+{ ObjectId } = require 'mongojs'
+{ ARTSY_URL, API_MAX, API_PAGE_SIZE } = process.env
+
+#
+# Schemas
+#
+schema = (->
+  id: @objectId()
+  partner_id: @objectId()
+  slug: @string().allow('', null)
```

Comment:
Example of converting string -> objectId in where function: https://github.com/kanaabe/positron/blob/master/api/apps/articles/model.coffee#L138 ...going to submit that patch to Joi-objectid some day!